### DISCUSSION TITLE
XAIOGYAN(ES2)基板のXAIO裏の穴部分を少し広げて、ESP32C3用のアンテナ線を基板裏に通せるようにしませんか？

### DISCUSSION THREAD

**Discussion by y-ohe** [algyan/XIAOGYAN#31] [Ideas]
XAIOGYAN基板の表(部品が実装される面)には、LED、LEDマトリックス、SW、エンコーダなどいろいろ載るため、アンテナを張り付ける場所がありません。
XAIGYAN(ES2)基板のXAIO裏の穴部分を少し広げて、ESP32C3用のアンテナ線を基板裏に通せるようにしませんか？

**Comment by matsujirushi** [comment]
アイデアありがとうございます！

通した後、アンテナをどこに貼り付ける想定ですか？？

**Reply by y-ohe** [reply]
アンテナは、サイズをちゃんと測ってないのですが、U2のLEDドライバとLEDマトリックスの裏側辺りを想定しています。
(U2の下側にはJP1とJP5があるので、ほぼLEDマトリックスの裏側辺りになるかも。)
なお、基板に直接貼ると、基板のGNDべたパターンなどと干渉(？)も懸念されるので、ちょっとギャップを設ける必要があるかもしれません。
なので、基板裏側の方のケースに張り付けるのもありと考えています。

**Reply by matsujirushi** [reply]
うーむ、、、
アンテナをどこに貼り付けるのが良いのかは、（ミスリードを避けたいので）実際に動かしてから決めさせてください。
PCBが手元に届いたら、ぜひ確認してください！

**Comment by matsujirushi** [comment]
ES2では、この穴にアンテナ線を結束バンドで固定できるようにしています。

![image](https://user-images.githubusercontent.com/8079404/220010353-8ce204b2-e52f-482d-b7cd-29a380b50ad7.png)

**Reply by y-ohe** [reply]
@matsujirushi さん
このスリットは、アンテナ線を結束するためのものだったんですね。
XAIOGYANを持ち運ぶ時のストラップひもを結ぶための穴かと思っていました(^_^;)
